src/patternfly/components/TabContent/examples/tab-content-example.hbs

```diff
@@ -0,0 +1,12 @@
+{{#> tab-content tab-content--IsActive="true" tab-content--attribute='id="tab1-panel" aria-labelledby="tab1"'}}
+  Panel 1
+{{/tab-content}}
+{{#> tab-content tab-content--attribute='id="tab2-panel" aria-labelledby="tab2"'}}
```

Comment:
It might be worth updating the `aria-controls=""` in the tab component to match the id of each of these sections.